web/src/services/config/index.ts

```diff
@@ -51,10 +51,12 @@ export type Config = {
 
 declare global {
   let __APP_VERSION__: string;
```

Comment:
not used? hash = version?

Reply:
It's package version as application, so it's not related to the hash.